modules/apps/transfer/keeper/genesis.go

```diff
@@ -1,13 +1,13 @@
 package keeper
 
 import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"context"
 
 	"github.com/cosmos/ibc-go/v9/modules/apps/transfer/types"
 )
 
 // InitGenesis initializes the ibc-transfer state and binds to PortID.
-func (k Keeper) InitGenesis(ctx sdk.Context, state types.GenesisState) {
+func (k Keeper) InitGenesis(ctx context.Context, state types.GenesisState) {
```

Comment:
do we want to update sig to return error here too? Most of functions invoked in here currently go the ol' panic route in most cases. Does sdk panic on returned error during genesis ops?

Reply:
assume it presents the error more gracefully now rather than the panic trace. 

I support propagating errors in a follow up PR. I chose not to do it here for obvious reasons

Reply:
https://github.com/cosmos/ibc-go/issues/7698